tests/config/fgcmBuildStarsTableHsc.py

```diff
@@ -27,6 +27,8 @@
 config.physicalFilterMap = HSC_FILTER_DEFINITIONS.physical_to_band
 config.doSubtractLocalBackground = True
 config.sourceSelector["science"].flags.bad.append("localBackground_flag")
+# Our test files do not have detect_isPrimary in the columns.
+config.sourceSelector["science"].doRequirePrimary = False
```

Comment:
Hhhmmm...hoping they don't have `sky_source`s either?!

Reply:
The test inputs that are used in {{testdata_jointcal}} for HSC were created before we added sky sources to the single-frame images (that long ago!)